components/image/stellarium/patches/01-cast-to-remove-compiler-ambiguity.patch

```diff
@@ -0,0 +1,11 @@
+--- /home/jgoerzen/src/stellarium-0.21.1/src/core/modules/LandscapeMgr.cpp	2021-06-18 22:14:42.000000000 +0000
++++ stellarium-0.21.1/src/core/modules/LandscapeMgr.cpp	2021-08-21 10:26:26.330117816 +0000
+@@ -132,7 +132,7 @@
+ 		{ dNE, Vec3f(-1.f,  1.f, 0.f) }, { dSE, Vec3f( 1.f,  1.f, 0.f) },
+ 		{ dSW, Vec3f( 1.f, -1.f, 0.f) }, { dNW, Vec3f(-1.f, -1.f, 0.f) }
+ 	};
+-	const float cp = 1.f/(1+sqrt(2));
++	const float cp = 1.f/(1+sqrt((float).2));
+ 	const float cn = -1.f*cp;
```

Comment:
What I read here is that you not only added a cast but also change from sqrt(2) to sqrt(0.2).